src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapSessionOptions.Linux.cs

```diff
@@ -57,8 +57,8 @@ public bool SecureSocketLayer
 
         public int ProtocolVersion
         {
-            get => GetPtrValueHelper(LdapOption.LDAP_OPT_VERSION).ToInt32();
-            set => SetPtrValueHelper(LdapOption.LDAP_OPT_VERSION, new IntPtr(value));
+            get => GetIntValueHelper(LdapOption.LDAP_OPT_VERSION);
+            set =>  SetIntValueHelper(LdapOption.LDAP_OPT_VERSION, value);
```

Comment:
Nit: extra spacing before method call.

Also, you don't need to force push every time for small changes. All changes in PRs will be squashed into one commit.

Reply:
@huoyaoyuan Fixed the extra spacing. Thanks for the feedback!